### PR TITLE
fixed:Search box doesn't render well in small views and dashboard responsiveness

### PR DESCRIPTION
**Pull request by theajmalrazaq**
i have fixed issue #83  Search box doesn't render well in small views please add hacktobefest label
image

### CODE REVIEW

**Comment by seveibar** [comment]
Can you show a screenshot with a mobile view

File: package.json [hunk 1]
```diff
@@ -99,6 +99,9 @@
     "vaul": "^0.9.9",
     "wouter": "^3.3.5"
   },
+  "prettier": {
```
Comment (seveibar) [inline package.json:102]: We use biome, can you remove this?

**Comment by seveibar** [comment]
Ah mobile view is different. This fix seems OK but i think we might want to improve the search bar there- its super small on md. I think maybe we could use a shadcn command modal when someone clicks the box on small screens?

**Comment by theajmalrazaq** [comment]
i think it seems perfect on mobile devices
image

**Review by seveibar — APPROVED** [review]
Nice!

**Comment by theajmalrazaq** [comment]
Screenshot_select-area_20241018202347
@seveibar i also made dashboard responsive on mobile kindly review and merge 
Screenshot_select-area_20241018202317

**Comment by seveibar** [comment]
I would run "bun format" after editing, the first pr for a person on github i have to manually approve workflows

**Comment by theajmalrazaq** [comment]
Hi @seveibar,
I sincerely apologize for the confusion caused. As a beginner, this is my first time contributing to a large open-source project, and I appreciate your understanding. Please kindly disregard any mistakes made.

**Comment by seveibar** [comment]
No worries! Just letting you know the first pr is always tricky and you might have to ping me more often 😁

**Comment by theajmalrazaq** [comment]
Haha, I think I’ve already pinged you more than 20 times! 😅 I really love your project and I’m excited to contribute more, especially with frontend bugs. You can definitely expect more pings from me!

**Review by seveibar — APPROVED** [review]
nice, future contributions will auto-run checks which will make things easier

**Pull request merged**